### DISCUSSION TITLE
How to get and display custom token such as USDT using Rainbowkit?

### DISCUSSION THREAD

**Discussion by cryptostar7** [rainbow-me/rainbowkit#1517] [Q&A] [answered]
Hello,

While developing a project using Rainbowkit, I faced the need to display a custom token.
As you know, the ConnectWallet component displays the network name, native coin name, and amount by default.
But now I need to display the token name and amount in my wallet.

Does Rainbowkit support that feature? So what should I do?

**Comment by DanielSinclair** [comment, marked as answer]
This is supported in Wagmi, yes. RainbowKit uses the `useBalance()` hook to fetch a user's gas token balance, but you can also use this hook to fetch a specific ERC-20 balance. In many cases, it is a better idea to rely on an indexer like Covalent or Alchemy for fetching a list of balances for a user.
```tsx
import { useBalance } from 'wagmi'
 
function App() {
  const balance = useBalance({
    address: '0xA0Cf798816D4b9b9866b5330EEa46a18382f251e',
    token: '0x1f9840a85d5af5bf1d1762f925bdaddc4201f984',
  })
}
```
https://wagmi.sh/react/hooks/useBalance#token-optional
